src/lib/utils.py

```diff
@@ -547,6 +547,15 @@ def dump_manifest(contents: t.Dict, manifest_path: t.Union[Path, str]):
     else:
         indent = 4
 
+    # Determine max line length preference
+    max_line_length: t.Optional[int]
+    if "max_line_length" in conf:
+        try:
+            max_line_length = int(conf.get("max_line_length"))
+            _yaml.width = max_line_length  # type: ignore # See https://sourceforge.net/p/ruamel-yaml/tickets/322/
+        except ValueError:
+            log.error("max_line_length EditorConfig variable is not valid")
```

Comment:
Error seems a bit severe. I'd make this a warning. And mypy annotations aside I think something like this would be clearer:

```suggestion
    if max_line_length := conf.get("max_line_length"):
        try:
            _yaml.width = int(max_line_length)  # type: ignore # See https://sourceforge.net/p/ruamel-yaml/tickets/322/
        except ValueError:
            log.warning("Ignoring invalid max_line_length %r", max_line_length)
```

Reply:
You are right, raising an error might be too much, a simple warning is ideal. 
Regarding your suggestions it looks good to me but do we want to drop `max_line_length: t.Optional[int]` ? I placed it in order to follow the same syntax as the others EditorConfig parameters

Reply:
I personally don't think it adds anything because we are doing our own runtime type-check, and `mypy` can surely infer that `int(max_line_length)` is of type `int`.

Reply:
Corrected the code following your recommandations